storybook/qmlTests/tests/tst_SwapSignModal.qml

```diff
@@ -42,7 +42,7 @@ Item {
             networkBlockExplorerUrl: "https://etherscan.io/"
 
             serviceProviderName: Constants.swap.paraswapName
-            serviceProviderURL: Constants.swap.termsAndConditionParaswapUrl
+            serivceProviderTandCUrl: Constants.swap.termsAndConditionParaswapUrl
```

Comment:
I'd keep them separate (ie both entries); I think the text "terms and conditions" should be the one that's clickable here, shouldn't it?

Reply:
Hey I had to add it back, cause after discussing with Ben both "Paraswap" and "Terms & Conditions" should be clickable. With the first one pointing to paraswap home page and second to t&c page.

Reply:
Yeah, that was actually what I thought too :)